src/components/layouts/pages/inspect/tree/Tree.js

```diff
@@ -0,0 +1,165 @@
+                    ruleSummaries={ruleSummaries}
+                    openedRule={openedRule}
+                    selectedCheck={selectedCheck}
+                    onRuleClick={onRuleClick}
+                    onCheckClick={onCheckClick}
+                />
+            </List>
+        </section>
+    );
+}
+
+// TODO: add Warnings
+function RuleList({ ruleSummaries, openedRule, selectedCheck, profile, onRuleClick, onCheckClick }) {
+    return ruleSummaries.map((rule, index) => {
+        const ruleTitle = getRuleTitle({ ...rule, profile });
+        return (
+            <Fragment key={index}>
+                <Tooltip
+                    placement="bottom-start"
+                    enterDelay={500}
+                    enterNextDelay={500}
+                    title={getRuleDescription({ ...rule, profile })}
+                >
+                    <ListItem button onClick={() => onRuleClick(index)} className="rule-item rule-item_error">
+                        {rule.checks.length ? openedRule === index ? <ExpandLess /> : <ChevronRightIcon /> : []}
+                        <ListItemText
+                            primary={
+                                <>
+                                    {ruleTitle}
+                                    {rule.checks.length && (
+                                        <Chip size="small" className="rule-item__chip" label={rule.checks.length} />
+                                    )}
+                                </>
+                            }
+                        />
+                    </ListItem>
+                </Tooltip>
+                {rule.checks.length && (
+                    <Collapse in={openedRule === index} timeout="auto" unmountOnExit>
+                        <List component="div" disablePadding>
+                            <CheckList checks={rule.checks} selectedCheck={selectedCheck} onCheckClick={onCheckClick} />
+                        </List>
+                    </Collapse>
+                )}
+            </Fragment>
+        );
+    });
+}
+
+function CheckList({ checks, selectedCheck, onCheckClick }) {
+    return checks.map(({ context }, index) => {
+        return (
+            <ListItem
+                key={index}
+                onClick={() => onCheckClick(context)}
+                selected={selectedCheck === context}
+                button
+                className="check-item"
+            >
+                <ListItemText primary={getCheckTitle(context, index, checks)} />
+            </ListItem>
+        );
+    });
+}
+
+// TODO: fix with future rule object
+function getRuleDescription({ profile, specification, clause, testNumber, description }) {
+    return errorMap[profile]?.[specification]?.[clause]?.[testNumber].DESCRIPTION || description;
+}
+
+function getRuleTitle({ profile, specification, clause, testNumber, test }) {
+    return errorMap[profile]?.[specification]?.[clause]?.[testNumber].SUMMARY || test;
```

Comment:
In case error map doesn't contain info let's fallback to 
```
`${specification}, clause ${clause}, test ${testNumber}`
```

Also format of translation file was changed a bit, now there is no profile level in it, instead it starts with specification right away.